quora-api/.gitignore

```diff
@@ -0,0 +1,10 @@
+Trello_quora.iml
```

Comment:
.gitignore is necessary for the whole project , it is not required for each module.

Reply:
Okay. Will remove it.